lib/src/rocket.rs

```diff
@@ -604,6 +604,19 @@ impl Rocket {
         self
     }
 
+    /// Get the managed state of type T, if it exists.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// let rocket = rocket::ignite().manage("example");
```

Comment:
This isn't a good example because it encourages the bad practice of managing a type not controlled by the library. Do something like this instead:

```rust
struct MyState(&'static str);

let rocket = rocket::ignite().manage(MyState("hello!"));
assert_eq!(rocket.state::<MyState>(), Some(MyState("hello!")));

let client = rocket::local::Client::new(rocket).expect("valid rocket");
assert_eq!(client.rocket().state::<MyState>(), Some(MyState("hello!")));
```